src/client/src/api/apps.rs

```diff
@@ -42,3 +123,97 @@ pub struct SlackApiAppsConnectionOpenRequest {}
 pub struct SlackApiAppsConnectionOpenResponse {
     pub url: SlackWebSocketsUrl,
 }
+
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackApiAppsManifestCreateRequest {
+    pub app_id: SlackAppId,
+
+    // HACK: This API requires a "json-encoded" string in a JSON object.
```

Comment:
I might be wrong here, but have you tried to use `serde_with` crate (already in deps) for  this?

https://docs.rs/serde_with/1.14.0/serde_with/json/index.html
something like:
> #[serde(with = "serde_with::json::nested")]

?

Reply:
Tested in a quick test, seems working fine for the same purpose.
The only thing about serde_with it is not included as default feature and can be enabled with:

> serde_with = { version = "1.14", features = ["json"] }

Reply:
Yes, it seems to fit perfectly for this situation!
I didn't know the crate, replaced with them @ 78cdcf5